shell-ui/theme.json

```diff
@@ -1,61 +1 @@
-{
```

Comment:
So we should drop the `ui-branding` ConfigMap in `salt/metalk8s/addons/ui/deployed/ui.sls.in` as well?